packages/components/pie-toast/src/toast.scss

```diff
@@ -92,18 +106,32 @@
   display: flex;
   align-items: center;
   gap: var(--dt-spacing-b);
-  padding: calc(var(--dt-spacing-03) / 2) 0;
+  padding: 6px var(--dt-spacing-a) 6px 0;
```

Comment:
Can we add a css variable to self document why this magic `6px` exists?

Reply:
it's per design specs and we don't have a token for it.